AUDIO/N_0xf1f8157b8c3f171c.md

```diff
@@ -1,17 +1,18 @@
 ---
 ns: AUDIO
 ---
-## _0xF1F8157B8C3F171C
+## SET_VEHICLE_STARTUP_REV_SOUND
 
 ```c
 // 0xF1F8157B8C3F171C 0x1C0C5E4C
-void _0xF1F8157B8C3F171C(Vehicle vehicle, char* p1, char* p2);
+void SET_VEHICLE_STARTUP_REV_SOUND(Vehicle vehicle, const char* p1, const char* p2);
```

Comment:
Does native gen handle const char?

Reply:
is used in GetVehicleClassFromName so i figured it did, if not i'll edit.

Reply:
I don't see const char being used in any actual natives, only in examples or native doc ports

Reply:
Will check and verify when I got time, seems like ur right on ur note though, never payed attention to const char or char

Reply:
👍 Also don't forget to add an alias for the old native see the [contribution guidelines](https://github.com/citizenfx/natives/blob/master/.github/CONTRIBUTING.md) for an example.